app/mailers/task_mailer.rb

```diff
@@ -0,0 +1,48 @@
+class TaskMailer < ApplicationMailer
+  def task_created_to_members(project, task)
+    @project = project
+    @task = task
+
+    mail(to: ProjectMembership.where(role: :member, project: project).map { |p| p.user.email })
```

Comment:
Can you use `has_many` association pros? `project.users.where(project_memberships: { role: :member }).pluck(:email)`